pkg/build/raw.go

```diff
@@ -24,13 +24,22 @@ var modifyRawImageTemplate string
 
 func (b *Builder) buildRawImage() error {
 	cmd := b.createRawImageCopyCommand()
-	err := cmd.Run()
-	if err != nil {
+	if err := cmd.Run(); err != nil {
 		return fmt.Errorf("copying the base image %s to the output image location %s: %w",
 			b.context.ImageDefinition.Image.BaseImage, b.generateOutputImageFilename(), err)
 	}
 
-	if err = b.writeModifyScript(); err != nil {
+	if err := b.modifyRawImage(b.generateOutputImageFilename(), true, true); err != nil {
+		// modifyRawImage will wrap the error, simply return it here
+		return err
+	}
+
+	return nil
+}
+
+func (b *Builder) modifyRawImage(imagePath string, includeCombustion, renameFilesystem bool) error {
+
```

Comment:
Nit: Leaving an empty line at the start of a function is considered bad practice.

Reply:
Fixed; you've pointed it out before and I normally check for it.